Services/include/services.h

```diff
@@ -43,7 +43,7 @@
 
 /* SERVICE SOCKETS */
 // HOUSEKEEPING SERVICE
-#define TC_HOUSEKEEPING_SERVICE 9
+#define TC_HOUSEKEEPING_SERVICE 17
```

Comment:
Why change this number?

Reply:
Had to. The added EPS config commands use this port number.